web/src/main/webapp/package.json

```diff
@@ -14,8 +14,8 @@
     "url": "git://github.com/mbudiu-vmw/hiero.git"
   },
   "dependencies": {
-    "@types/d3": "^4.9.0",
-    "d3": "^4.0.5",
+    "@types/d3": "^4.10.0",
```

Comment:
Do we have to manually upgrade, or will webpack cause these packages to be upgraded automatically?

Are you using any functionality which is new in d3 version 10?